src/cpp/session/resources/schema/user-prefs-schema.json

```diff
@@ -200,10 +205,16 @@
                 "tabSet2": ["Files", "Plots", "Packages", "Help", "Viewer"],
                 "hiddenTabSet": [],
                 "console_left_on_top": false,
-                "console_right_on_top": true
+                "console_right_on_top": true,
+                "additional_source_columns": 0
             },
             "description": "Layout of panes in the RStudio workbench."
         },
+        "enable_additional_columns": {
+           "type": "boolean",
+           "default": false,
+           "description": "Temporary flag to enable additional source columns."
```

Comment:
Recommend adding a `title` to this pref so that it can be toggled from the Command Palette (you'll also need to regenerate prefs afterward)

Reply:
Done!